src/SFA.DAS.Payments.Monitoring.Metrics.Data/DcMetricsDataContext.cs

```diff
@@ -286,7 +424,16 @@ public async Task<List<ProviderNegativeEarningsLearnerDcEarningAmounts>> GetNega
         {
             using (await BeginTransaction(cancellationToken))
             {
-                var result = await AllNegativeEarnings.FromSqlRaw(BaseDcEarningsQuery + LearnerNegativeEarnings, new SqlParameter("@collectionperiod", collectionPeriod)).ToListAsync(cancellationToken);
+                var result = new List<ProviderNegativeEarningsLearnerDcEarningAmounts>();
+                if (academicYear == 2526)
```

Comment:
introduces technical debt and requires a follow up ticket to fix after R03.  Would be better to have code that used the new sql from 2526 onwards.